README.md

```diff
@@ -91,7 +122,11 @@ Define a function called `doSomething`. It should take one argument, called
 
 Your Answer:
 ```js
-// write code here
+function doSomething(thingToDo) {
+  console.log(thingToDo + " is done.")
+};
+
```

Comment:
not quite...the way you have this function set it up, it requires one argument `"thingToDo"`. When calling `doSomething(); `below we would need to pass in one argument as well.

Here is the solution for this question:
```
function doSomething(thingToDo){
  thingToDo();
}
doSomething(function(){
  alert("Yay!");
});
```